src/olympia/editors/helpers.py

```diff
@@ -710,6 +716,38 @@ def confirm_auto_approved(self):
         AddonApprovalsCounter.increment_for_addon(addon=self.addon)
         self.log_action(amo.LOG.CONFIRM_AUTO_APPROVED)
 
+    def reject_multiple_versions(self):
+        """Reject a list of versions."""
+        # self.version and self.files won't point to the versions we want to
+        # modify in this action, so set them to None before finding the right
+        # versions.
+        self.version = None
+        self.files = None
+        for version in self.data['versions']:
+            files = version.files.all()
+            self.set_files(amo.STATUS_DISABLED, files, hide_disabled_file=True)
+            self.log_action(amo.LOG.REJECT_VERSION,
+                            version=version, files=files)
+        self.addon.update_status()
+        self.data['version_numbers'] = u', '.join(
+            unicode(v.version) for v in self.data['versions'])
+
+        # Send the email to the developer. We need to pass down a version for
+        # the ActivityLogToken generation to work - it might still result in
+        # an expired token if it's not the latest version available for the
+        # add-on, but it's the best we can do.
```

Comment:
I could do that, but there is no guarantee than `self.version` will have anything to do with the versions we're rejecting. I guess for the typical expected usage scenario (a reviewer finds an add-on that should be completely rejected after several of its versions have been auto-approved, rejects them all, including the latest one) both solutions will work. I can change it if you prefer.